packages/@headlessui-react/src/components/popover/popover.tsx

```diff
@@ -461,6 +461,8 @@ let Button = forwardRefWithAs(function Button<TTag extends ElementType = typeof
         dispatch({ type: ActionTypes.ClosePopover })
         state.button?.focus() // Re-focus the original opening Button
       } else {
+        event.preventDefault()
+        event.stopPropagation()
```

Comment:
This change totally borks us from utilizing events.
A `<Popover.Button as="div">` wrapping an element now blocks any event bubbling and we cannot actually react to these events for an item wrapped in a tooltip popover.

This is a clear inversion of control problem.  I do not believe that the decision to preventDefault and stopPropagation on an event is the sole responsibility of headlessui to decide.

A user could supply an onClick function that calls preventDefault or stopPropagation if they so choose to.  But with this code change the end users no longer have a choice in this.

We were forced to roll back to 1.5.0